doc/manuals/quick_start_guide_new.md

```diff
@@ -0,0 +1,211 @@
+
+The interesting part is the "id" field, in the above example `frb`.
+
+[Top](#top)
+
+###<a name="section3.3"></a>Step 3: Upload some data to HDFS
+You can upload your own data to your HDFS space using the WebHDFS RESTful API listening on TCP/14000 port of the <i>Storage Endpoint</i>.
+
+Let's start by creating a new directory (`testdir`) in our HDFS user space (in this example, `hdfs:///user/frb`). `curl` has been used as REST client:
+
+    $ curl -X PUT "http://storage.cosmos.lab.fiware.org:14000/webhdfs/v1/user/frb/testdir?op=MKDIRS&user.name=frb" -H "X-Auth-token: 3azH09G1PdaGmgBNODLOtxy52f5a00" | python -m json.tool
+    {"boolean": true}
+
+Now, it is time to upload some local file (`testdata.txt`) to the fresh new directory we have created (please observe the verbose option `-v` has been used):
+
+```
+$ cat testdata.txt 
+luke,tatooine,jedi,25
+leia,alderaan,politician,25
+solo,corellia,pilot,32
+yoda,dagobah,jedi,275
+vader,tatooine,sith,50
+```
+
+```
+$ curl -v -X PUT -T testdata.txt "http://storage.cosmos.lab.fiware.org:14000/webhdfs/v1/user/frb/testdir/testdata.txt?op=CREATE&user.name=frb" -H "Content-Type: application/octet-stream" -H "X-Auth-token: 3azH09G1PdaGmgBNODLOtxy52f5a00"
+*   Trying 195.235.93.174...
+* Connected to storage.cosmos.lab.fiware.org (195.235.93.174) port 14000 (#0)
+> PUT /webhdfs/v1/user/frb/testdir/testdata.txt?op=CREATE&user.name=frb HTTP/1.1
+> Host: storage.cosmos.lab.fiware.org:14000
+> User-Agent: curl/7.43.0
+> Accept: */*
+> Content-Type: application/octet-stream
+> X-Auth-token: 3azH09G1PdaGmgBNODLOtxy52f5a00
+> Content-Length: 118
+> Expect: 100-continue
+> 
+< HTTP/1.1 100 Continue
+* We are completely uploaded and fine
+< HTTP/1.1 307 Temporary Redirect
+< X-Powered-By: Express
+< Access-Control-Allow-Origin: *
+< Access-Control-Allow-Methods: HEAD, POST, GET, OPTIONS, DELETE
+< Access-Control-Allow-Headers: origin, content-type, X-Auth-Token, Tenant-ID, Authorization
+< server: Apache-Coyote/1.1
+< set-cookie: hadoop.auth="u=frb&p=frb&t=simple&e=1460661599535&s=Uzn+QdUaqGpZqXsoyNb9cCUuJtU="; Version=1; Path=/; Expires=Thu, 14-Apr-2016 19:19:59 GMT; HttpOnly
+< location: http://dev-fiwr-svc-01.tid.es:14000/webhdfs/v1/user/frb/testdir/testdata.txt?op=CREATE&user.name=frb&data=true
+< Content-Type: application/json; charset=utf-8
+< content-length: 0
+< date: Thu, 14 Apr 2016 09:19:59 GMT
+< connection: close
+< 
+* Closing connection 0
+```
+
+The above command just has started the uploading operation. As can be seen, the WebHDFS service redirects us to the following location:
+
+    location: http://dev-fiwr-svc-01.tid.es:14000/webhdfs/v1/user/frb/testdir/testdata.txt?op=CREATE&user.name=frb&data=true
+    
+That's becasue the first operation only created the new `hdfs:///user/testdir/testdata.txt` HDFS file in the Namenode; not it is time to upload the data bytes to the Datanodes, and that's achieved by PUTting again the local `testdata.txt` file in the redirection URL:
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/pull/171/commits/5befe509562cafde2a6ce71ed81f812506e8c235